inputs-optional.tf

```
@@ -72,3 +72,14 @@ variable "source_server_id" {
   default     = null
   description = "Source server ID for point in time restore. Only used when create mode is set to PointInTimeRestore"
 }
+
+variable "jenkins_AAD_objectId" {
```

what about this so it's not specific to Jenkins?
```suggestion
variable "admin_user_object_id" {
```

---

done